src/gui/src/browserWidget.cpp

```diff
@@ -563,7 +569,7 @@ void BrowserWidget::makeRowItems(QStandardItem* item,
   }
 
   QString text
-      = locale.toString(disp_area, 'f', 3) + " " + units + "m\u00B2";  // m2
+      = QString::number(disp_area, 'f', 3) + " " + units + "m\u00B2";  // m2
```

Comment:
delete  "m\u00B2" to header, change header to "Area/m\u00B2"

Reply:
The units are dynamically set to um^2 or mm^2 depending on the value.  Putting it in the header could result in very large values that are hard to read.

Reply:
conditional code?

Reply:
No longer necessary as the display and sort values are separated.